evaluator/src/main/kotlin/Evaluator.kt

```diff
@@ -39,7 +39,12 @@ import org.ossreviewtoolkit.utils.scripting.ScriptRunner
 class Evaluator(
     ortResult: OrtResult = OrtResult.EMPTY,
     licenseInfoResolver: LicenseInfoResolver = OrtResult.EMPTY.createLicenseInfoResolver(),
-    licenseClassifications: LicenseClassifications = LicenseClassifications()
+    licenseClassifications: LicenseClassifications = LicenseClassifications(),
+
+    /**
+     * The date and time of the evaluation to be used by policy rules as the current date and time.
```

Comment:
The repetition of "data and time" sounds a bit odd, IMO.

Actually, I'm thinking about renaming the parameter to just `time` as the "evaluation" bit in `evaluationTime` really sounds very much like what `startTime` is.

And if it's just generically called "time", I think we could even drop the docs, as it's then completely up to the implementation *what* time is it and how to use it in the rules.